meta/ee-requirements.txt

```diff
@@ -1,2 +1,3 @@
 docker
-docker-compose
+# do not add docker-compose python package as a depedency as the module
+# is unmaintained and it is now a go project and called by docker directly.
```

Comment:
This needs a) a changelog fragment, and b) a second PR with a deprecation changelog fragment (to be merged and released soon). This one can only be merged once we start preparing 3.0.0 (which isn't planned right now **at all**).

Reply:
Meanwhile I removed this collection from creator-ee.